docs/RELEASE.md

```diff
@@ -0,0 +1,62 @@
+# Processes
+
+The following processes are necessary to actualize our releases. Each process has a *Cadence* on which it must execute and an *Responsible* that is responsible for autonomously doing so and reporting back any error in the RelEng<sup>1</sup> channel.
+
+## Mainline Release
+
+Cadence: every two weeks. Responsible: Release Team.
+
+This process aims to release the `stable` branch as a *Mainline* release every two weeks. It needs to start three days before the proposed release deadline to allow leeway for unexpected issues. This process should eventually be automated.
+
+### Steps
+
+1. [ ] Check if process [Clobbering](#clobbering) needs to happen and do so first, if that is the case.
+1. [ ] Check out the latest commit of `stable`.
+2. [ ] Verify all CI checks of that commit.
+3. [ ] Announce that commit as cutoff *Mainline* for a release in the General<sup>2</sup> chat.
```

Comment:
As there should be way less merges to `stable`, I would say that the release is actually done as some pr against the `stable` branch. IMO this gives more visibility and also makes it more clear on what is happening.

Reply:
> IMO, we should make the process less depended on posting something into some internal channels. This would be totally opaque to the outside and not that good.

Yea right. The channel pings would be mostly for us to react to issues and monitoring, but having it as comments in a MR is probably better.